providers/isend-sms/jest.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
```

Comment:
To resolve the Jest test issue [here](https://github.com/novuhq/novu/actions/runs/7260977977/job/19781519293?pr=5000#step:5:71), a module name mapper is required due to Jest's lack of support for ES modules.

An example in another provider is available [here](https://github.com/novuhq/novu/blob/1950a191ffb351875c7d79a16237ed9e23000226/providers/resend/jest.config.js#L6).
```suggestion
  testEnvironment: 'node',
  moduleNameMapper: {
    axios: 'axios/dist/node/axios.cjs',
  },
```